app/Http/Controllers/RetrievingRecords.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+
+class RetrievingRecords extends Controller
```

Comment:
The controller should be more specific. If it is for handling resources, you can rename it to `ResourcesController`.